### ISSUE TITLE
npm support?

### ISSUE THREAD

**Issue by jamesfzhang** [bitwalker/keys.js#6]
Hi, great library, thanks for building it. Do you plan on adding npm support?

**Comment by bitwalker** [comment]
Sure, I can do that. I should probably refactor things a bit to bring it more up to date, but for now I'll see if I can't just get a package definition pushed and get it up on npmjs.com.

**Comment by jamesfzhang** [comment]
Thanks @bitwalker, much appreciated!

**Comment by bitwalker** [comment]
Done! https://www.npmjs.com/package/keys.js

**Comment by jamesfzhang** [comment]
@bitwalker How do you require keys?

I've tried these, no luck

```js
require('keys.js');
require('keys');
```

**Comment by bitwalker** [comment]
@jamesfzhang Let me take a look, it should just work. Are you using ES6/2015-style require/import, or require.js/some variant?

**Comment by jamesfzhang** [comment]
I'm using ES6 style via https://github.com/browserify-rails/browserify-rails

**Comment by bitwalker** [comment]
@jamesfzhang cool, I'll take a look and get back to you soon

**Comment by bitwalker** [comment]
@jamesfzhang Found the issue, I can now require the script in Node, so it should work with browserify.

You can do either:

```javascript
var Keys = require('keys.js');

// or

import {Key, Combo, Bindings} from 'keys.js';
```

Let me know if you are still running into problems!

**Comment by jamesfzhang** [comment]
@bitwalker I still can't seem to require it via browserify-rails. I suspect it might be related to the `.` in the library name? If I move the src file from `node_modules` into my assets pipeline, I can require it relatively. That'll do for now, thanks!

**Comment by bitwalker** [comment]
Hmm, I wouldn't think so as there are quite a few libs with the `something.js` naming scheme, perhaps I'm missing something else. I'll try and find some time to play with it in browserify.

**Comment by bitwalker** [comment]
@jamesfzhang I found the problem, I needed to add `main` to my package.json file. Just did a quick test and it appears to be working properly now. Let me know if you are still having trouble (published a new version 1.0.2 btw).

**Comment by jamesfzhang** [comment]
@bitwalker Works perfectly now, thanks a lot! Cheers.
